Sources/NFCPassportReader/TagReader.swift

```diff
@@ -113,7 +113,7 @@ public class TagReader {
         
         let data = oidBytes + keyTypeBytes
             
-        let cmd = NFCISO7816APDU(instructionClass: 00, instructionCode: 0x22, p1Parameter: 0xC1, p2Parameter: 0xA4, data: Data(data), expectedResponseLength: 256)
+        let cmd = NFCISO7816APDU(instructionClass: 00, instructionCode: 0x22, p1Parameter: 0xC1, p2Parameter: 0xA4, data: Data(data), expectedResponseLength: -1)
```

Comment:
Why do we set this to -1? Is it true that the Le field is not present for all cases we use this function?

Reply:
Yes, the response data is always absent for all cases we use this function, which is only for PACE. It's specified in Doc9303 part 11 - 4.4.4.1. Bjørn mentioned that he had done the same error - _most_ documents allow the APDU to not be 100% correct and by the standard, which is why it has mostly worked before, but some documents don't allow any deviance. 

There are other functions we should probably also change to -1 Le (such as sendMSEKAT (p11 - 6.2.4.1), sendMSESetATIntAuth (p11 - 6.2.4.2), and maybe others) but I haven't seen any errors from these functions, and don't have enough specimen to test with now